SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -8,17 +8,30 @@
 %define config_source %{SOURCE1}
 %endif
 
+%ifarch aarch64
+%global __provides_exclude_from %{_libdir}/debug/.build-id/
+%define arch arm64
+%define archdir arm64
+%define config_source %{SOURCE2}
+%endif
+
+# Generated by update script
+%global pkgrel 1000
+%global kernelver 5.15.98.mshv1
+%global gitrev d77ef975
+
 Summary:        Linux Kernel for Kata UVM
 Name:           kernel-uvm
-Version:        5.15.48.1
-Release:        9%{?dist}
+Version:        %{kernelver}
+Release:        %{pkgrel}.g%{gitrev}%{?dist}
 License:        GPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Kernel
-URL:            https://github.com/microsoft/CBL-Mariner-Linux-Kernel
-Source0:        https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner-2/%{version}.tar.gz#/kernel-%{version}.tar.gz
+URL:            https://microsoft.visualstudio.com/DefaultCollection/LSG/_git/linux-dom0
+Source0:        kernel-mshv-%{version}.tar.gz
 Source1:        config
```

Comment:
We still need a config file because there is no default uvm config in this release, it was added on 99. So next release, we'll be able to use the default and get rid of the one we maintain, as we did for kernel-mshv